src/components/Pool/SortByDropdown/SortByDropdown.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useState } from 'react'
+import { ChevronDown } from 'react-feather'
+import { useTranslation } from 'react-i18next'
+import { Flex, Text } from 'rebass/styled-components'
+import styled, { useTheme } from 'styled-components'
+
+import { LIQUIDITY_SORTING_TYPES } from '../../../constants'
+import { useOnClickOutside } from '../../../hooks/useOnClickOutside'
+import Popover from '../../Popover'
+
+export function SortByDropdown({
+  sortBy,
+  onSortByChange,
+}: {
+  sortBy: string
+  onSortByChange: (sortBy: string) => void
+}) {
+  const popoverRef = useRef()
+  const { t } = useTranslation()
+  const [isOpen, setIsOpen] = useState(false)
+  const theme = useTheme()
+  useOnClickOutside(popoverRef, () => setIsOpen(false))
```

Comment:
I would also add one suggestion which I think its easy fix.

Usually on tables like this if you were to click on tvl, 24volume or apy it would sort based on that. Thats very intuitive behaviour and if you were to click on it once again it would sort in opposite direction.
@zamli @0xVenky Wdyt? @adamazad 
![Screenshot 2022-06-15 at 14 18 52](https://user-images.githubusercontent.com/33226956/173825146-196c7e67-fb37-42e2-bebd-2b577d46e211.png)